src/main/webapp/src/app/nav-bar/nav-bar.component.spec.ts

```diff
@@ -14,20 +12,19 @@ describe('NavBarComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      imports: [ RouterTestingModule ],
+      imports: [ 
+        RouterTestingModule.withRoutes(routes)
+      ],
       declarations: [ NavBarComponent ],
-      providers: [
-        NavBarComponent
-      ]
+      providers: [  ]
     })
     .compileComponents();
   });
 
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [
-        CommonModule,
-        RouterTestingModule.withRoutes(routes)
+        
```

Comment:
As this is empty, I believe you should be fine removing the whole TestBed.configureTestingModule function?